packages/scene-composer/src/components/panels/CommonPanelComponents.tsx

```diff
@@ -214,7 +214,7 @@ export interface ExpandableInfoSectionProps {
 export const ExpandableInfoSection: React.FC<React.PropsWithChildren<ExpandableInfoSectionProps>> = ({
   title,
   withoutSpaceBetween = false,
-  defaultExpanded = true,
+  defaultExpanded = false,
```

Comment:
are we sure we want to default collapse the sections in the right side component details panel?

Reply:
Good catch! Fixed.